runelite-client/src/main/java/net/runelite/client/plugins/gamescaling/GameScalingConfig.java

```diff
@@ -23,14 +23,14 @@
  * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package net.runelite.client.plugins.stretchedfixedmode;
+package net.runelite.client.plugins.gamescaling;
 
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
 
-@ConfigGroup("stretchedfixedmode")
-public interface StretchedFixedModeConfig extends Config
+@ConfigGroup("gamescaling")
```

Comment:
FWIW this will break existing stretched fixed mode config for users

Reply:
I think it's worth it.

Reply:
And searching for "Stretched Fixed Mode" in config sidebar still returns this plugin, so it should also not confuse users v much